src/newSeed/seedConfig.ts

```diff
@@ -78,6 +78,7 @@ export interface ISeedConfig {
   tokenDetails: ITokenDetails,
   contactDetails: IContactDetails,
   seedDetails: ISeedDetails,
+  clearState: () => void,
```

Comment:
does this appear in JSON.stringify?

Reply:
No, a copy of the seedConfig excluding the function is stringified

Reply:
Stringify ignores the function